packages/vsx-registry/src/browser/style/index.css

```diff
@@ -17,6 +17,7 @@
 :root {
   --theia-vsx-extension-icon-size: calc(var(--theia-ui-icon-font-size)*3);
   --theia-vsx-extension-editor-icon-size: calc(var(--theia-vsx-extension-icon-size)*3);
+  --theia-vsx-extension-header-height: 150px;
```

Comment:
how do you get 150?

Reply:
Thanks for the comment @akosyakov 
I had originally used this value since VSCode seems to have a fixed header size of 166px but also includes a tabbar (which we do not have). I have removed this hard coded value and instead calculated the heights using JS from the deferred HTMLElements inside `vsx-extension-editor.tsx `.